net/libfab/libfab.c

```diff
@@ -1085,11 +849,12 @@ static inline struct m0_fab__ep *libfab_ep(struct m0_net_end_point *net)
  * a matching endpoint.
  */
 static bool libfab_ep_cmp(struct m0_fab__ep *ep, const char *name,
-			  uint64_t *ep_name_n)
+			  struct m0_net_ip_addr *addr)
 {
+	struct m0_net_ip_addr *fep = &ep->fep_name;
 	return (name != NULL &&
-		strcmp(ep->fep_name_p.fen_str_addr, name) == 0) ||
-	       (name == NULL && *ep_name_n == ep->fep_name_n);
+		strcmp(ep->fep_name.nia_p, name) == 0) ||
+	        (name == NULL && m0_net_ip_addr_eq(fep, addr, true));
```

Comment:
For optimization comparison can be changes e.g. so that non-string related comparison will be execute first

( addr != NULL && m0_net_ip_addr_eq(fep, addr, true) ) ||
( name != NULL && strcmp(ep->fep_name.nia_p, name) == 0) )

Reply:
Removed this function in latest commit.